src/ui/public/index_patterns/_field.js

```diff
@@ -40,10 +40,10 @@ export default function FieldObjectProvider(Private, shortDotsFilter, $rootScope
 
     const indexed = !!spec.indexed;
     const scripted = !!spec.scripted;
-    const sortable = spec.name === '_score' || ((indexed || scripted) && type.sortable);
-    const filterable = spec.name === '_id' || scripted || (indexed && type.filterable);
     const searchable = !!spec.searchable || scripted;
     const aggregatable = !!spec.aggregatable || scripted;
+    const sortable = spec.name === '_score' || ((indexed || scripted || aggregatable) && type.sortable);
```

Comment:
It looks like `aggregatable` implies `scripted || aggregatable`, so `scripted` could be left out here.